docs/framework/get-started/system-requirements.md

```diff
@@ -20,7 +20,7 @@ manager: "wpickett"
 
 # .NET Framework system requirements
 
-The tables in this topic provide the hardware, operating system, and software requirements for the .NET Framework 4.5 and its point releases (4.5.1 and 4.5.2), the [!INCLUDE[net_v46](../../../includes/net-v46-md.md)] and its point releases (4.6.1 and 4.6.2), and the .NET Framework 4.7. Development environments that enable you to develop apps for the .NET Framework have a separate set of requirements.
```

Comment:
The entries for "[!INCLUDE" are not showing correctly on the webpage.

Reply:
It looks fine to me @vivmishra. Are you checking at https://review.docs.microsoft.com/en-us/dotnet/framework/get-started/system-requirements?branch=pr-en-us-3415?

Reply:
Ok. Maybe I was missing the part that this needs to be seen in docs rather than viewing the md file directly.